tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/merge.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import PageRegion
+
+
+class Merge(PageRegion):
```

Comment:
You're right, the page region was a bad call. We usually use page regions in order to split a page into smaller parts. Here it's already been done in the iFrame. So if we use `Base` instead of `PageRegion`, we'll still only see the Merge panel, as the rest is outside the iFrame. 

So let's use `Base` here. Sorry, for the wrong information